MiniApp/Classes/MiniAppLister.swift

```diff
@@ -14,4 +14,21 @@ internal class MiniAppLister {
             }
         }
     }
+
+    func fetchMiniApp(miniAppId: String, apiClient: MiniAppClient, completionHandler: @escaping (Result<MiniAppInfo, Error>) -> Void) {
+
+        apiClient.getMiniApp(miniAppId) { (result) in
+            switch result {
+            case .success(let responseData):
+                guard let decodeResponse = ResponseDecoder.decode(decodeType: Array<MiniAppInfo>.self, data: responseData.data), let miniApp = decodeResponse.first else {
+                    return completionHandler(.failure(NSError.invalidResponseData()))
+                }
+
+                return completionHandler(.success(miniApp))
```

Comment:
Two different styles of spacing/formatting around `return`, one here and one above.